src/App.js

```diff
@@ -11,33 +11,128 @@ class App extends React.Component {
+      // render states (player1CurrentScore and player1Total)
+      this.setState({
+        roundWinner: 1,
+        player1CurrentScore: this.state.player1CurrentScore + 1,
+        player1Total: this.state.player1Total + 1,
+      });
+    } else if (newCurrCards[1].rank > newCurrCards[0].rank) {
+      // render states (player2CurrentScore and player2Total)
+      this.setState({
+        roundWinner: 2,
+        player2CurrentScore: this.state.player2CurrentScore + 1,
+        player2Total: this.state.player2Total + 1,
+      });
+    } else {
+      // if no winner
+      this.setState({ roundWinner: 0 });
+    }
+
+    this.setState((prevState) => ({
       // Remove last 2 cards from cardDeck
-      cardDeck: state.cardDeck.slice(0, -2),
-      // Deal last 2 cards to currCards
-      currCards: state.cardDeck.slice(-2),
+      cardDeck: prevState.cardDeck.slice(0, -2),
+      // pop newCurrCards into currCards array
+      currCards: newCurrCards,
+      gameStart: true,
+      // update gameRound
+      gameRound: prevState.gameRound + 1,
     }));
   };
 
+  // reset game
+  resetGame = () => {
+    this.setState({
+      cardDeck: makeShuffledDeck(),
+      currCards: [],
+      hasGameStarted: false,
+      roundWinner: null,
+      player1Total: 0,
+      player2Total: 0,
+    });
+  };
+
   render() {
-    const currCardElems = this.state.currCards.map(({ name, suit }) => (
-      // Give each list element a unique key
-      <div key={`${name}${suit}`}>
-        {name} of {suit}
-      </div>
-    ));
+    // display cards
+    const currCardElements = this.state.currCards.map(
+      ({ name, suit }, index) => (
+        // Give each list element a unique key
+        <div key={`${name}${suit}`}>
+          {/* retrieve card images */}
+          <img
+            src={require(`../src/assets/${name}Of${suit}.png`)}
+            alt="cardImage"
+          />
+          <p>
+            Player {index + 1}'s card is {name} of {suit}
+          </p>
+        </div>
+      ),
+    );
+    // round winner's messages (2 expectations)
+    const roundWinnerMessage = this.state.roundWinner
+      ? `Player ${this.state.roundWinner} won this round.`
+      : `No winner this round. The game continues!`;
+
+    // individual winner's message
+    // if player1CurrentScore > player2CurrentScore, player 1 wins, else player 2 wins
+    const winnerMessage =
+      this.state.player1CurrentScore > this.state.player2CurrentScore
+        ? `Player 1 wins the game`
+        : `Player 2 wins the game`;
+
+    // players' current and total scores
+    const player1CurrentRoundScore = `Player 1's current round score is: ${this.state.player1CurrentScore}`;
+    const player2CurrentRoundScore = `Player 2's current round score is: ${this.state.player2CurrentScore}`;
+    const player1TotalScore = `Player 1 won a total of: ${this.state.player1Total} rounds.`;
+    const player2TotalScore = `Player 2 won a total of: ${this.state.player2Total} rounds.`;
```

Comment:
Since these are static, we might as well just include them in the HTML directly.